towhee/dag/graph_repr.py

```diff
@@ -40,15 +57,33 @@ def operators(self) -> Dict[str, OperatorRepr]:
     def dataframes(self) -> Dict[str, DataframeRepr]:
         return self._dataframes
 
-    def from_yaml(self, yaml: str):
+    def from_yaml(self, src: str):
```

Comment:
Use staticmethod, and return repr object

Reply:
fixed